models/news.models.js

```diff
@@ -75,4 +75,8 @@ exports.updateArticleVotes = (articleId, incVotes) => {
 };
 
 exports.selectUsers = () => 
-  db.query(`SELECT * FROM users;`).then(({rows: users}) => users);
\ No newline at end of file
+  db.query(`SELECT * FROM users;`).then(({rows: users}) => users);
+
+exports.deleteCommentById = commentId => 
+  db.query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *;`, [commentId])
+  .then(({rows}) => rows);
```

Comment:
Seem as though though dont actually use the rows, i think it would work the same without this .then, and just use the .then from the controller.